.github/workflows/xe-experimental-vm-test.yml

```diff
@@ -0,0 +1,35 @@
+name: "[experimental] VM-based integration testing, ping @Xe if it fails"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - 'main'
+  release:
+    types: [ created ]
+  schedule:
+    - cron: '00 6,18 * * *'
```

Comment:
I forgot this magic syntax. Comment alongside it?

Reply:
I added a comment explaining it as well as a link to crontab.guru